gluten-core/src/main/scala/org/apache/gluten/extension/injector/RuleInjector.scala

```diff
@@ -19,12 +19,12 @@ package org.apache.gluten.extension.injector
 import org.apache.spark.sql.SparkSessionExtensions
 
 /** Injector used to inject query planner rules into Spark and Gluten. */
-class RuleInjector {
-  val spark: SparkInjector = new SparkInjector()
+class RuleInjector(val extensions: SparkSessionExtensions) {
```

Comment:
```suggestion
class RuleInjector(val spark: SparkSessionExtensions) {
```

Reply:
In the world of Spark, Spark usually means to SparkSession.
For consistent, let's use `exts`.